airbyte-integrations/connectors/source-sendgrid/source_sendgrid/schemas/unsubscribe_groups.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
Can you add "additionalProperties": "true"? This will make it so that the validation won't fail in the future when new fields are added.